Sensors.cpp

```diff
@@ -56,6 +70,26 @@ void Sensors::init(int pms_type, int pms_rx, int pms_tx) {
     am2320Init();
 }
 
+void Sensors::am2320Init() {
+    am2320.begin();  // temp/humidity sensor
+}
+
+void Sensors::sht31Init() {
+    sht31.begin(0x44);  // temp/humidity sensor
```

Comment:
this address doesn't change? if yes, it's ok.

Reply:
No, the address don{t change but in the examples is used the (0x44)